### PR TITLE
DR-1162 Add button for start billing process

### PR DESCRIPTION
**Pull request by mvillaseco**
# Overview
 Add payment workflow.

# Changes
 - Add function to call service.
 - Add service to post required data and redirect to reports after.

### CODE REVIEW

File: src/wwwroot/controllers/BillingCtrl.js [hunk 1]
```diff
@@ -129,7 +130,14 @@
       vm.cc.parsedCcNumber = utils.replaceAllCharsExceptLast4(vm.cc.number);
       vm.secCode.ParsedNumber = utils.replaceAllCharsExceptLast4(vm.secCode.number);
       vm.viewExpDate = form.expDate.$viewValue;
+    }
 
+    function submitBillingPayment() {
+      var paymentData = { plan : planName, creditCard: { number: vm.cc.number, securityCode: vm.secCode.number, expiration: vm.expDate} };
+      settings.billingPayment(paymentData)
```
Comment (andresmoschini) [inline src/wwwroot/controllers/BillingCtrl.js]: Add return in order to keep the promise:

```js
return settings.billingPayment(paymentData)
```

File: src/wwwroot/controllers/BillingCtrl.js [hunk 2]
```diff
@@ -129,7 +130,14 @@
...
+      .then(function() {
+        $location.path('/');
```
Comment (andresmoschini) [inline src/wwwroot/controllers/BillingCtrl.js]: Should we redirect to my-plan? and show the new data?

**Comment by andresmoschini** [comment]
👍

File: src/wwwroot/services/settings.js [hunk 3]
```diff
@@ -137,5 +138,19 @@
 
       return plansCache;
     }
+
+    function billingPayment(paymentData) {
+      var url = RELAY_CONFIG.baseUrl
+        + '/account/'
+        + auth.getAccountName()
+        + '/aggrements/';
```
Comment (nreal) [inline src/wwwroot/services/settings.js]: The last `/` is not necessary

**Pull request merged**